recipes/aliview/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "AliView" %}
+{% set version = "1.30" %}
+{% set sha256 = "3ed6af471a2e5c77355c1a0d06a31e44478a84ee93cb15b7295861549b5332f7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/AliView/AliView/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  run_exports:
+    - {{ pin_subpackage("aliview", max_pin="x") }}
+  noarch: generic
+  number: 0
+  script: |
+    mvn clean compile install package -DskipTests
```

```suggestion
    mvn clean package -DskipTests
```
* there is no need of `compile` `goal because `package` implies it
* there is no need of `install` goal because it just installs it at `~/.m2/repository/`
* usually `verify` goal is preferred than `package` but since you ignore the unit tests than I assume you want to ignore then integration tests too